crates/sqlbuiltins/src/functions/scalars.rs

```diff
@@ -0,0 +1,994 @@
+    reverse
+}
+document! {
+    "Extract a substring from the end of a string with the given length.",
+    "right('hello', 2)",
+    right
+}
+document! {
+    "Pad a string to the right to a specified length with a sequence of characters.",
+    "rpad('hello', 10, '12')",
+    rpad
+}
+document! {
+    "Remove all spaces from the end of a string.",
+    "rtrim(' hello ')",
+    rtrim
+}
+document! {
+    "Split a string on a delimiter and return the nth field.",
+    "split_part('hello.world', '.', 2)",
+    split_part
+}
+document! {
+    "Split a string on a delimiter and return an array of the fields.",
+    "string_to_array('hello world', ' ')",
+    string_to_array
+}
+document! {
+    "Returns true if the first string starts with the second string, false otherwise.",
+    "starts_with('hello world', 'hello')",
+    starts_with
+}
+document! {
+    "Find the position of the first occurrence of a substring in a string.",
+    "strpos('hello world', 'world')",
+    strpos
+}
+document! {
+    "Extract a substring from a string with the given start position and length.",
+    "substr('hello', 2, 2)",
+    substr
+}
+document! {
+    "TODO",
+    "TODO",
+    to_hex
+}
+document! {
+    "Replace all occurrences of a set of characters in a string with a new set of characters.",
+    "translate('hello', 'el', '12')",
+    translate
+}
+document! {
+    "Remove all spaces from the beginning and end of a string.",
+    "trim(' hello ')",
+    trim
+}
+document! {
+    "Convert a string to uppercase.",
+    "upper('hello')",
+    upper
+}
+document! {
+    "Generate a random UUID.",
+    "uuid()",
+    uuid
+}
+document! {
+    "Returns true if the first string matches the second string as a regular expression, false otherwise.",
+    "regexp_match('hello world', 'hello')",
+    regexp_match
+}
+document! {
+    "Replace all occurrences of a substring in a string with a new substring using a regular expression.",
+    "regexp_replace('hello world', 'hello', 'goodbye')",
+    regexp_replace
+}
+document! {
+    "Returns the current timestamp.",
+    "now()",
+    now
+}
+document! {
+    "Returns the current date.",
+    "current_date()",
+    current_date
+}
+document! {
+    "Returns the current time in UTC",
+    "current_time()",
+    current_time
+}
+document! {
+    "Returns the date binned to the specified interval.",
+    "date_bin('15 minutes', TIMESTAMP '2022-01-01 15:07:00', TIMESTAMP '2020-01-01)",
+    date_bin
+}
+document! {
+    "Returns the date truncated to the specified unit.",
+    "date_trunc('day', '2020-01-01')",
+    date_trunc
+}
+document! {
+    "Returns the specified part of a date.",
+    "date_part('year', '2020-01-01')",
+    date_part
+}
+document! {
+    "Converts a string to a timestamp (Timestamp<ns, UTC>). Alias for `TIMESTAMP <string>`.",
+    "to_timestamp('2020-09-08T12:00:00+00:00')",
+    to_timestamp
+}
+document! {
+    "Converts a string to a timestamp with millisecond precision (Timestamp<ms, UTC>)",
```

Comment:
the strings are inconsistent about weather or not they are a sentence or not. Some end in  a period, and some don't, I think being consistent is good styple.